Anatoly Shlom/grid/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Paris</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/grid.css">
+</head>
+<body>
+    <main class="widget">
+        <!--article class="widget-data"-->
+        <!-- Интересный вопрос, как в моем случае сохранить сетку и не сломать семантику?-->
```

Comment:
The same as I've added a comment to your previous PR, `article` is't the most appropriate tag here.

Reply:
Thank you) Now i got the point